src/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ endfunction()
 if(AMALGAMATION_BUILD)
 
   add_library(duckdb SHARED "${PROJECT_SOURCE_DIR}/src/amalgamation/duckdb.cpp")
+  set_target_properties(duckdb PROPERTIES VERSION ${DUCKDB_VERSION_NUMBER} SOVERSION ${DUCKDB_MAJOR_VERSION})
```

Comment:
```suggestion
  set_target_properties(duckdb PROPERTIES VERSION ${DUCKDB_VERSION_NUMBER} SOVERSION ${DUCKDB_MAJOR_VERSION}.${DUCKDB_MINOR_VERSION})
```

Reply:
Would something like this work as well? We are still moving fast, I don't think we guarantee 1.3.x and 1.4.x libraries to be compatible.

Reply:
Sure, that seems to work fine. I've changed the commit, the resulting files look like this now:
```
lrwxrwxrwx  1 builder abuild   16 Jul 18 12:04 libduckdb.so -> libduckdb.so.1.3
lrwxrwxrwx  1 builder abuild   18 Jul 18 12:04 libduckdb.so.1.3 -> libduckdb.so.1.3.2
-rwxr-xr-x  1 builder abuild  64M Jul 18 12:04 libduckdb.so.1.3.2
-rw-r--r--  1 builder abuild  63M Jul 18 12:04 libduckdb_static.a
```